.env.staging.example

```
@@ -0,0 +1,231 @@
+# =============================================================================
+# APPLICATION CONFIGURATION
+# =============================================================================
+
+# Environment
+NODE_ENV=staging
+VITE_APP_ENV=staging
+VITE_APP_VERSION=1.0.0-staging
+
+# Domain Configuration
+VITE_APP_DOMAIN=https://staging.prmcms.com
+VITE_APP_NAME=PRMCMS (Staging)
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)